2019-01-29 04:45:45,527 INFO  execution-log - 	Initializing control map with map # MASTER
----------------
2019-01-29 04:45:45,533 INFO  execution-log - Initializing Test Automation
----------------
2019-01-29 04:45:45,537 INFO  execution-log - 	target browser: Chrome
----------------
2019-01-29 04:45:45,542 INFO  execution-log - TC # : 2, Downloading test case from DB
----------------
2019-01-29 04:45:45,569 INFO  execution-log - TC # : 2, Getting control map :MASTER from DB
----------------
2019-01-29 04:45:45,600 INFO  execution-log - TC # : 2, Control map retrieval completed
----------------
2019-01-29 04:45:45,616 INFO  execution-log - TC # : 2, Getting test data from DB
----------------
2019-01-29 04:45:45,631 INFO  execution-log - TC # : 2, Test data retrieval completed
----------------
2019-01-29 04:45:45,631 INFO  execution-log - TC # : 2, Parsing commands
----------------
2019-01-29 04:45:45,631 INFO  execution-log - Loading captured values from persistence.
----------------
2019-01-29 04:45:45,647 INFO  execution-log - TC # : 2, Started executing test case in the VMTESTAPP1 environment (http://vam-qa.cloudapp.net:6363/pages/login.html)
----------------
2019-01-29 04:45:45,663 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 1 )
----------------
2019-01-29 04:45:45,678 INFO  execution-log - TC # : 2, Starting iteration #1
----------------
2019-01-29 04:45:45,678 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:45,741 INFO  execution-log -  Beginning InvokeApplicationCommand ( Target :  )
----------------
2019-01-29 04:45:46,674 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:46,683 INFO  execution-log -  Beginning PageCommand ( CurrentPage : Login, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-01-29 04:45:46,690 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:46,697 INFO  execution-log -  Beginning LoadTestDataItemCommand ( CurrentPage : Login, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-01-29 04:45:46,708 INFO  execution-log - [Sequence No - 1] Beginning ValidateExistStatusCommand ( LogicalFieldName : User Name, LogicalFieldValue : True )
----------------
2019-01-29 04:45:46,724 INFO  execution-log - [Sequence No - 1]	 Command completed.
----------------
2019-01-29 04:45:46,724 INFO  execution-log - [Sequence No - 2] Beginning ActionCommand ( LogicalFieldName : User Name, LogicalFieldValue : Click )
----------------
2019-01-29 04:45:46,850 INFO  execution-log - [Sequence No - 2]	 Command completed.
----------------
2019-01-29 04:45:46,860 INFO  execution-log - [Sequence No - 3] Beginning TestDataCommand ( LogicalFieldName : User Name, LogicalFieldValue : Jack )
----------------
2019-01-29 04:45:46,963 INFO  execution-log - [Sequence No - 3]	 Command completed.
----------------
2019-01-29 04:45:46,973 INFO  execution-log - [Sequence No - 4] Beginning ValidateExistStatusCommand ( LogicalFieldName : Password, LogicalFieldValue : True )
----------------
2019-01-29 04:45:46,994 INFO  execution-log - [Sequence No - 4]	 Command completed.
----------------
2019-01-29 04:45:47,004 INFO  execution-log - [Sequence No - 5] Beginning TestDataCommand ( LogicalFieldName : Password, LogicalFieldValue : Smit@123 )
----------------
2019-01-29 04:45:47,134 INFO  execution-log - [Sequence No - 5]	 Command completed.
----------------
2019-01-29 04:45:47,145 INFO  execution-log - [Sequence No - 6] Beginning ValidateEnableStatusCommand ( LogicalFieldName : Login_Button, LogicalFieldValue : True )
----------------
2019-01-29 04:45:47,175 INFO  execution-log - [Sequence No - 6]	 Command completed.
----------------
2019-01-29 04:45:47,175 INFO  execution-log - [Sequence No - 7] Beginning ActionCommand ( LogicalFieldName : Login_Button, LogicalFieldValue : Click )
----------------
2019-01-29 04:45:47,290 INFO  execution-log - [Sequence No - 7]	 Command completed.
----------------
2019-01-29 04:45:47,296 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:47,301 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 2 )
----------------
2019-01-29 04:45:47,319 ERROR execution-log - 	Due to an unexpected error, the test run will be aborted. Please contact your administrator
----------------
2019-01-29 04:45:51,600 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 1 )
----------------
2019-01-29 04:45:51,600 INFO  execution-log - TC # : 2, Starting iteration #1
----------------
2019-01-29 04:45:51,600 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:51,616 INFO  execution-log -  Beginning InvokeApplicationCommand ( Target :  )
----------------
2019-01-29 04:45:57,961 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:57,973 INFO  execution-log -  Beginning PageCommand ( CurrentPage : Login, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-01-29 04:45:57,978 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:57,983 INFO  execution-log -  Beginning LoadTestDataItemCommand ( CurrentPage : Login, FlowIdentifier : 1, DiIndicator : DI )
----------------
2019-01-29 04:45:57,998 INFO  execution-log - [Sequence No - 1] Beginning ValidateExistStatusCommand ( LogicalFieldName : User Name, LogicalFieldValue : True )
----------------
2019-01-29 04:45:58,022 INFO  execution-log - [Sequence No - 1]	 Command completed.
----------------
2019-01-29 04:45:58,033 INFO  execution-log - [Sequence No - 2] Beginning ActionCommand ( LogicalFieldName : User Name, LogicalFieldValue : Click )
----------------
2019-01-29 04:45:58,158 INFO  execution-log - [Sequence No - 2]	 Command completed.
----------------
2019-01-29 04:45:58,170 INFO  execution-log - [Sequence No - 3] Beginning TestDataCommand ( LogicalFieldName : User Name, LogicalFieldValue : Jack )
----------------
2019-01-29 04:45:58,268 INFO  execution-log - [Sequence No - 3]	 Command completed.
----------------
2019-01-29 04:45:58,280 INFO  execution-log - [Sequence No - 4] Beginning ValidateExistStatusCommand ( LogicalFieldName : Password, LogicalFieldValue : True )
----------------
2019-01-29 04:45:58,310 INFO  execution-log - [Sequence No - 4]	 Command completed.
----------------
2019-01-29 04:45:58,319 INFO  execution-log - [Sequence No - 5] Beginning TestDataCommand ( LogicalFieldName : Password, LogicalFieldValue : Smit@123 )
----------------
2019-01-29 04:45:58,440 INFO  execution-log - [Sequence No - 5]	 Command completed.
----------------
2019-01-29 04:45:58,449 INFO  execution-log - [Sequence No - 6] Beginning ValidateEnableStatusCommand ( LogicalFieldName : Login_Button, LogicalFieldValue : True )
----------------
2019-01-29 04:45:58,482 INFO  execution-log - [Sequence No - 6]	 Command completed.
----------------
2019-01-29 04:45:58,485 INFO  execution-log - [Sequence No - 7] Beginning ActionCommand ( LogicalFieldName : Login_Button, LogicalFieldValue : Click )
----------------
2019-01-29 04:45:58,604 INFO  execution-log - [Sequence No - 7]	 Command completed.
----------------
2019-01-29 04:45:58,611 INFO  execution-log - 	 Command completed.
----------------
2019-01-29 04:45:58,617 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 2 )
----------------
2019-01-29 04:45:58,631 ERROR execution-log - 	Due to an unexpected error, the test run will be aborted. Please contact your administrator
----------------
2019-01-29 04:46:04,886 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 1 )
----------------
2019-01-29 04:46:04,886 ERROR execution-log - 	Due to an unexpected error, the test run will be aborted. Please contact your administrator
----------------
2019-01-29 04:46:13,105 INFO  execution-log -  Beginning StartIterationCommand ( Iterations : 1 )
----------------
2019-01-29 04:46:13,120 ERROR execution-log - TC # : 2, Due to an unexpected error, the test run will be aborted. Please contact your administrator
----------------
2019-01-29 04:46:13,120 ERROR execution-log - TC # : 2, System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at VM.Platform.TestAutomationFramework.UIAutomation.Commands.Workflow.StartIterationCommand.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.UIAutomation\Commands\Workflow\StartIterationCommand.cs:line 135
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandLogger.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandLogger.cs:line 74
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandDecorator.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandDecorator.cs:line 22
   at VM.Platform.TestAutomationFramework.Core.Decorators.CommandErrorHandler.Execute(TestRunContext context) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\Decorators\CommandErrorHandler.cs:line 30
----------------
2019-01-29 04:46:15,449 ERROR execution-log - TC # : 2, Framework failed to run the test due to a test configuration error.
	System.Threading.ThreadAbortException: Thread was being aborted.
   at VM.Platform.TestAutomationFramework.Core.TestCommandRunner.Execute(TestRunContext testRunContext) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\TestCommandRunner.cs:line 107
   at VM.Platform.TestAutomationFramework.Core.SingleTestCaseRunner.ExecuteSingleTestCase(Int32 testCaseId, String targetEnvironment, String repositiory, Int32 Retry) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\SingleTestCaseRunner.cs:line 167
----------------
2019-01-29 04:46:15,605 ERROR execution-log - An error has caused test case #2 to abort. Proceeding to the next test case.
System.Threading.ThreadAbortException: Thread was being aborted.
   at VM.Platform.TestAutomationFramework.Core.SingleTestCaseRunner.ExecuteSingleTestCase(Int32 testCaseId, String targetEnvironment, String repositiory, Int32 Retry) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\SingleTestCaseRunner.cs:line 189
   at VM.Platform.TestAutomationFramework.Core.TestAutomation.Execute(Int32 testCaseId, Int32 Retry, String targetEnvironment) in C:\Users\VMQApractice\Desktop\v4Dev_Updated\v4Dev_Updated\VMTestAutomation\Src\Framework.Core\TestAutomation.cs:line 203
----------------
